_data/projects/misc/conda-forge-root.yml

```
@@ -0,0 +1,41 @@
+name: conda-forge ROOT
+description: CERN's ROOT on Conda-Forge.
+url: https://github.com/conda-forge/root-feedstock
+readme: https://github.com/conda-forge/root-feedstock/blob/master/README.md
+affiliated: true
+# repo: only if different from url
+longdescription: |
+
+    CERN's [ROOT][] framework traditionally has been difficult to install --
+    several hours on a single core, with many dependencies.  The situation is
+    better than it used to be, it is easier to build and available in more
+    places, but Python was still a challenge; ROOT is linked to just one
+    version of Python, and the one you get with pre-built ROOT can often be the
+    wrong one.  And, if you use the [Anaconda Python distribution][Anaconda],
```

I think we should promote miniconda here instead of anaconda as it's almost always the right choice for people in HEP.
```suggestion
    wrong one.  And, if you use the [Conda Python distribution][miniconda],
```